src/root/Multiplexer/index.js

```diff
@@ -105,12 +109,20 @@ function Multiplexer(props) {
     }
   }, [allRegionsResponse, getAllRegions]);
 
+  React.useEffect(() => {
+    if (!disasterTypes.fetching && !disasterTypes.fetched && !disasterTypes.cached) {
+      getDisasterTypes();
+      console.info('all disasterTypes not found in cache, requesting all disasterTypes');
+    }
+  });
+
   const pending = React.useMemo(() => (
     (languageResponse.fetching || !languageResponse.fetched)
     || (allCountriesResponse.fetching || (!allCountriesResponse.cached && !allCountriesResponse.fetched))
     || (allRegionsResponse.fetching || (!allRegionsResponse.cached && !allRegionsResponse.fetched))
     || (!skipUserDetails && (userResponse.fetching || (!userResponse.cached && !userResponse.fetched)))
-  ), [allCountriesResponse, userResponse, allRegionsResponse, skipUserDetails, languageResponse]);
+    || (!disasterTypes || (!disasterTypes.cached && !disasterTypes.fetched))
```

Comment:
not sure but shouldn't it be 
```js 
|| (disasterTypes.fetching || (!disasterTypes.cached && !disasterTypes.fetched))
```

Reply:
@frozenhelium Somehow `fetching` and `fetched` were always `false` (except after I added `fetched: true` to the Multiplexer part).

Reply:
That's okay, because we would not start the request if it is in the cache. So, `fetching` and `fetched` will mostly be `false` and `cached` will be true in that case